### ISSUE TITLE
The validate tool should check for any unbalanced tags

### ISSUE THREAD

**Issue by ritave** [MetaMask/SIPs#35]
Any code tags, or any kind of brackets should be balanced. Unbalanced tags signify that there's some kind of formatting error

**Comment by Mrtenz** [comment]
> [@Mrtenz](https://github.com/Mrtenz) close?

I don't think we ever implemented this, but it's also not a high priority issue, and can be caught through PR reviews.

**Comment by ajeetd** [comment]
Closing based on above comment.
